src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/template-editor/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths
+    ignorePath: paths.build + '/template-editor' // ensures proper relative paths
+  }))
+  .pipe(gulp.dest(paths.build + '/template-editor'));
+});
+
+gulp.task('template-editor', function () {
+  return gulp.src([
+    paths.app + '/template-editor/adapterService.js',
```

Comment:
What do you think about including all files in the folder `paths.app + '/template-editor/'`?

Reply:
Ok 👍